### ISSUE TITLE
Does not honour default application for some files

### ISSUE THREAD

**Issue by eugeneia** [elementary/files#854]
```
io.elementary.files 4.1.2
```

Since a couple of days ago files no longer remembers my default applications for some types of files. When clicking them it asks me to select an application, and there is no default (first item from the top) in the context menu.

![screenshot from 2019-01-09 11-18-00](https://user-images.githubusercontent.com/4933566/50893465-351b1180-1401-11e9-8b4e-b0d09d4f9ed1.png)

However, ticking the “Set as default” check mark does not result in files remembering the selected application. On opening the file another time I get presented with the application selection window, and the default entry in the context menu does not exist.

![screenshot from 2019-01-09 11-18-42](https://user-images.githubusercontent.com/4933566/50893480-3cdab600-1401-11e9-9648-69911606efb7.png)

Interestingly, I can’t set a default application for the file type in the “Properties” window like I can with seemingly unaffected file types.

![screenshot from 2019-01-09 11-19-08](https://user-images.githubusercontent.com/4933566/50893484-4106d380-1401-11e9-8e9e-b956a059db44.png)

Setting the default application from the command line does not help either:
```
$ xdg-mime default emacs26.desktop text/markdown
$ xdg-mime query default text/markdown
emacs26.desktop
```

**Comment by peteruithoven** [comment]
Thanks for the report! I'm afraid this is a duplicate of https://github.com/elementary/files/issues/836

**Comment by eugeneia** [comment]
:+1: Somehow missed that issue.

**Comment by citrusui** [comment]
Finally fixed with Files v4.1.3
